build.sbt

```diff
@@ -22,7 +22,6 @@ val excludedPackages = Seq(
   "testOnly.*",
   "com.kenshoo.play.metrics*.*",
   ".*repositories.*",
-  ".*LanguageSwitchController",
   ".*GuiceInjector",
   ".*models.Mode",
   ".*filters.*",
```

Comment:
exclude package has too many elements here which should not be here, some examples are: 
-   ".*standardError*.*",
-   ".*main_template*.*",
-   "views.html.*",
- config
- repositories
- viewmodels
and so on if you can take 10 mins here to clean them up, the easy way to start and learn about this is just remove some and make sure in newly generate scoverage html report there are no new auto generated classes like ReverseRoutes...